style.css

```diff
@@ -16,11 +16,25 @@ body {
   justify-content: space-between;
 }
 
-.nav-list {
+@media (min-width: 728px) {
+  .nav-bar {
+    width: 1440px;
```

Comment:
- [Optional] Great job with the exact design, however, it is best to use the Relative units because they scale relative to the window size or the parent. They specify the length, which is relative to another length property.👌👍 such as `em`, `rem` ...